test/appengine/nose.cfg

```diff
@@ -1,4 +0,0 @@
-[nosetests]
-cover-min-percentage=0
-with-gae=1
-gae-application=test/appengine/app.yaml
```

Comment:
Yeah, these look like things that will require manual intervention to reproduce.